src/window.js

```diff
@@ -56,10 +55,6 @@ export default function Window({
     type,
   });
 
-  window.connect("style-updated", () => {
-    setHighlightColour();
-  });
```

Comment:
Could you elaborate why this or a GTK4 equivalent isn't required?

It's not part of https://docs.gtk.org/gtk4/migrating-3to4.html

Reply:
This code was used to update the color of the highlighter when the theme is updated.

As you can see in https://github.com/sonnyp/Commit/blob/main/data/screenshot.png

For now and because I have to look into the new dark mode you can replace this signal handler with a simple call to `setHighlightColour()`.